alcs-api/src/comment/comment.controller.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { classes } from '@automapper/classes';
+import { AutomapperModule } from '@automapper/nestjs';
+import { createMock } from '@golevelup/nestjs-testing';
+import { ForbiddenException, NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ClsService } from 'nestjs-cls';
+import { CommentProfile } from '../common/automapper/comment.automapper.profile';
+import { mockKeyCloakProviders } from '../common/utils/test-helpers/mockTypes';
+import { CommentController } from './comment.controller';
+import { Comment } from './comment.entity';
+import { CommentService } from './comment.service';
+
+describe('CommentController', () => {
+  let controller: CommentController;
+  let mockCommentService;
+
+  let comment;
+  let user;
+
+  let request;
+
+  beforeEach(async () => {
+    mockCommentService = createMock<CommentService>();
+
+    user = {
+      name: 'Bruce Wayne',
+      email: 'fake-email',
+    };
+
+    request = {
+      user,
+    };
+
+    comment = new Comment({
+      body: 'body',
+      madeBy: user as any,
```

Comment:
obviously not a required thing madeBy => author?

Reply:
author is a much better term! Not sure why my brain couldnt come up with anything lol. Will change.

Reply:
Done!